terraform/main.tf

```
@@ -52,6 +52,25 @@ resource "azurerm_shared_image" "linux-sig" {
   }
 }
 
+resource "azurerm_key_vault" "vault" {
+  name                        = "packer-acctest-vault"
```

Right now if two developers are working on the plugin, or if CI is running when a developer is working, this can cause conflict, you should use the resource prefix for resources that have unique name across subscription, which I think key vaults are at least subscription wide for unique names, since we ran into that error before

```suggestion
  name                        = "${var.resource_prefix}-pkr-test-vault"
```